bedrock/firefox/templates/firefox/new/quantum/base.html

```diff
@@ -40,6 +40,10 @@
 {% block site_header %}{% endblock %}
 
 {% block content %}
+{% if LANG == 'en-US' and switch('netneutrality-12-2017') %}
```

Comment:
Can do the lang check in the switch call (here and in other templates):

`{% if switch('netneutrality-12-2017', ['en-US']) %}`